.gitignore

```diff
@@ -12,3 +12,9 @@ target/
 /yarn.lock
 /node_modules
 /.metadata/
+
+# Bazel stuff
+/bazel-bin
+/bazel-closure-compiler
+/bazel-out
+/bazel-testlogs
```

Comment:
Running `./build_test.sh` created bazel output directories that were added to the git tree by default. These lines prevent that.